MediaPlayer/tests/MediaSlider-specs.js

```diff
@@ -0,0 +1,69 @@
+import '@testing-library/jest-dom';
+import {fireEvent, render, screen} from '@testing-library/react';
+
+import MediaSlider from '../MediaSlider';
+
+const blur = (slider) => fireEvent.blur(slider);
+const focus = (slider) => fireEvent.focus(slider);
+const activate = (slider) => fireEvent.keyUp(slider, {keyCode: 13});
+const keyDown = (keyCode) => (slider) => fireEvent.keyDown(slider, {keyCode});
+
+const leftKeyDown = keyDown(37);
+const rightKeyDown = keyDown(39);
+
+describe('MediaSlider', () => {
+	test('should fire `onFocus` when focused', () => {
+		const handleFocus = jest.fn();
+
+		render(
+			<MediaSlider data-testid="mediaslider-id" onFocus={handleFocus} />
+		);
+
+		const slider = screen.getByTestId('mediaslider-id');
+		focus(slider);
+
+		expect(handleFocus).toBeCalled();
+	});
+	test('should fire `onBlur` on blur', () => {
```

Comment:
Please add an empty line between each test.

Reply:
added it